cmd/stop.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/semaphoreci/cli/cmd/pipelines"
+	"github.com/spf13/cobra"
+)
+
+var stopCmd = &cobra.Command{
+	Use:   "stop [KIND]",
+	Short: "Stop resource execution.",
+	Long:  ``,
+	Args:  cobra.RangeArgs(1, 2),
+}
+
+var StopPplCmd = &cobra.Command{
+	Use:     "pipelines [id]",
```

Comment:
I would suggest to use singular for this `pipeline [id]` and plural `pipelines` in the aliases.

The reason would be that `Use` part is displayed in the help screens of the CLI. `stop pipeline [id]` is better in this context then `stop pipelines [id]` IMHO.